adding cardinality description for event hub trigger

kencharos opened the pull request:
I added description of cardinality and sample code.
I found cardinality property for Javascript in [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-best-practices#receive-messages-in-batch-whenever-possible). And I tried it.

azurecla commented:
Hi __@AliakseiT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

kencharos committed:
adding cardinality sample trigger

I added description of cardinality and sample code.
I found cardinality property for Javascript in [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-best-practices#receive-messages-in-batch-whenever-possible). And I tried it.
